tests/legacy/test_crypto.py

```diff
@@ -1,31 +1,17 @@
 """Test Pack and Unpack."""
```

Comment:
Should stay here; I'm treating the crypto module in the legacy folder as a "do not touch, backwards compat only" thing since it went out in a release already.

Reply:
And it mimics exactly the dependency it replaces, aries-staticagent-python